Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1996,6 +1998,14 @@ extension BrowserViewController: ContextMenuHelperDelegate {
             popoverPresentationController.permittedArrowDirections = .Any
         }
 
+        if (count(dialogTitle!) >= maxDialogUrlLength) {
```

Comment:
I don't see an implicit proof that `dialogTitle` is never null, so let's add a line before:

```
dialogTitle = dialogTitle ?? ""
```

Hardly perfect, but it'll do.